src/movies.js

```diff
@@ -1,16 +1,103 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+
+function getAllDirectors(movieArray) {
+    let directors = movieArray.map(function(item){
+        return item.director;
+    })
+}
```

Comment:
For the function to do something, you will need to return the directors array you created.